common/src/Model/Node.cpp

```diff
@@ -43,8 +43,8 @@ namespace TrenchBroom {
         m_selected(false),
         m_childSelectionCount(0),
         m_descendantSelectionCount(0),
-        m_visibilityState(VisibilityState::Visibility_Inherited),
-        m_lockState(LockState::Lock_Inherited),
+        m_visibilityState(VisibilityState::Visibility_Shown),
```

Comment:
Yeah, `SetVisiblityCommand::reset` is still correct, although it might be better named `SetVisiblityCommand::inherit`.

> And why not default to inherited?

My thinking was, this was the best place to put it to get the "newly created brushes/entities are always visible, even if created in locked/hidden layers" behaviour... 
But I just saw that `MapDocument::addNodes` already has this:

```
const std::vector<Model::Node*> addedNodes = collectChildren(nodes);
ensureVisible(collectChildren(nodes));
```

So maybe I can get rid of this change to the Node constructor. I'll just need to add a similar `ensureUnlocked`. Will give it a try

Reply:
This worked out, so I undid the Node.cpp change